keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -153,6 +154,11 @@ type LeaderMigrationConfiguration struct {
 	// LeaderName is the name of the resource under which the controllers should be run.
 	LeaderName string `json:"leaderName"`
 
+	// ResourceLock indicates the resource object type that will be used to lock
+	// Should be either "leases" or "endpoints", but not both
```

Comment:
Both is referred to as "endpointsleases" (Eg. https://github.com/kubernetes/kubernetes/blob/a16df75344c6036730e1fba42b25c64c704b1494/staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go#L37)

Reply:
Verbosely add  this to type comment.